python/nav/models/sql/changes/sc.05.10.0003.sql

```diff
@@ -0,0 +1,5 @@
+-- Add new deviceState alert
+INSERT INTO alerttype (eventtypeid,alerttype,alerttypedesc) VALUES
+  ('deviceState','deviceDeletedChassis','The device has been removed as a chassis.');
+  INSERT INTO alerttype (eventtypeid,alerttype,alerttypedesc) VALUES
+  ('deviceState','deviceDeletedModule','The device has been removed as a module.');
```

Comment:
Come to think of it, I think I forgot to look at this in your previous PR.  5.10 is the current stable release, this will only hit 5.11, so the migration script should be named differently...

Reply:
I just found out that I was unable to merge 5.10.x back into master because the OUI changescript and a bugfix from 5.10 had conflicting changescript names.  I had to forcibly rename the OUI script on master, so there may be breakage for devs that do not wipe and re-create their postgres databases.

Reply:
yeah fuked up the filename quite hard for PowerSupplyAndFan aswell